src/modules/public/blacklistAccess/actions.ts

```diff
@@ -0,0 +1,37 @@
+import {
+    SEND_ACCESS_TOKEN_DATA,
+    SEND_ACCESS_TOKEN_ERROR,
+    SEND_ACCESS_TOKEN_FETCH,
+} from './constants';
+
+export interface SendAccessTokenFetch {
+    type: typeof SEND_ACCESS_TOKEN_FETCH;
+    payload: {
+        whitelink_token: string;
```

Comment:
Maybe better to put names:
whitelink_token -> allowed_link_token
According to replacement of problematic racist terminology (to avoid changing in the future)